ross/fluid_flow/fluid_flow.py

```diff
@@ -449,87 +458,92 @@ def calculate_coefficients(self, direction=None):
                     "Error: The given parameters create a rotor that is not inside the stator. "
                     "Check parameters and fix accordingly."
                 )
+        return c1, c2, c0w
 
-    def mounting_matrix(self):
+    def mounting_matrix(self, c1, c2, c0w):
         """This function assembles the matrix M and the independent vector f.
         Examples
         --------
         >>> my_fluid_flow = fluid_flow_example()
-        >>> my_fluid_flow.mounting_matrix()
-        >>> my_fluid_flow.M # doctest: +ELLIPSIS
-        array([[...
+        >>> c1, c2, c0w = my_fluid_flow.calculate_coefficients()
+        >>> my_fluid_flow.mounting_matrix(c1, c2, c0w)# doctest: +ELLIPSIS
+        (array([[...
```

Comment:
The new arguments (c1, c2, c0w) and outputs (M, f) should be added to the docstring.